debian-bullseye/verible.dockerfile

```diff
@@ -0,0 +1,52 @@
+# Authors:
+#   Unai Martinez-Corral
+#   Patrick Richer St-Onge
+#
+# Copyright 2022 Unai Martinez-Corral <unai.martinezcorral@ehu.eus>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ARG REGISTRY='gcr.io/hdl-containers/debian/bullseye'
+
+#---
+
+FROM $REGISTRY/build/build AS build
+
+RUN apt-get update -qq \
+    && apt-get -y install --no-install-recommends \
+        gpg \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > /usr/share/keyrings/bazel-archive-keyring.gpg \
+    && echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+
+RUN apt-get update -qq \
+    && apt-get -y install --no-install-recommends \
+        bazel \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/chipsalliance/verible /tmp/verible \
+    && cd /tmp/verible \
+    && bazel run -c opt //:install -- /opt/verible
```

Comment:
Please, use `/usr/local` as the default PREFIX. In other words, `bazel run -c opt //:install -- /opt/verible/usr/local` and then `COPY --from=build /opt/verible /` in the last stage below.

As you can see in https://github.com/hdl/containers/runs/6517925087?check_suite_focus=true#step:3:426, the test of pkg/verible will show the content in `/usr/local`.

Reply:
Ok, I think it's good now

Reply:
It seems that some of the "binaries" do have extension `.sh`: https://github.com/hdl/containers/runs/6529877383?check_suite_focus=true#step:3:479. Is that expected?

Reply:
Oh right there are three helper scripts around `verible-verilog-format`: `git-verible-verilog-format.sh`, `verible-transform-interactive.sh` and  `verible-verilog-format-changed-lines-interactive.sh` (documented [here](https://github.com/chipsalliance/verible/tree/master/verilog/tools/formatter)). They are part of Verible releases on GitHub, so I guess I should just add them to the tests?

Reply:
I merged this PR, because it has been waiting long enough.

Upon further thingking, those scripts (at least 2 of them) are to be used interactively, which can be really painful to do in CI. Therefore, I think it's ok not to include them in the tests. Nevertheless, should you want to do further improvements (either here or in hdl/smoke-tests), fell free to propose follow-up PRs.

Thanks a lot for your contribution and patience!